adoc/TRD-SLES-SAP-HA-automation-quickstart-cloud.adoc

```diff
@@ -442,7 +442,7 @@ endif::[]
 
 If successful, the output will be the public IP addresses for the cluster nodes.
 
-image::complete.png[width=470]
+image::images/src/jpg/TRD_SLES-SAP-HA-automation-quickstart-cloud-s3-bucket-aws-automation-complete[width=470]
```

Comment:
And once again: 

image::TRD_SLES-SAP-HA-automation-quickstart-cloud-s3-bucket-aws-automation-complete[width=470]